### PR TITLE
Updated SQS Processor blueprints

### PR DESCRIPTION
**Pull request by austoonz**
*Issue #, if available:*
There is no issue related to this.

*Description of changes:*
1. Cleaned up wording in the SQS readme file.

2. The original SQS Processor blueprint included a number of comments to provide a single script that could process a number of different event sources. This made the sample confusing and hard to follow. This PR cleans up the original SQS Processor to only provide the base code sample.

3. In addition, and to resolve the removal of the additional detail, this PR adds the same S3Event samples from [PR 380](https://github.com/aws/aws-lambda-dotnet/pull/380) so they're available from the S3Event or SQSProcessor blueprints.

4. This PR also adds an SNS to SQS sample.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by normj** [comment]
Same comment as the S3 PR. Put each template in a separate folder with its own README file and an entry in the `ps-lambda-blueprint-manifest.json` file.

**Comment by austoonz** [comment]
Norm - I have updated this PR as requested.

1. The SQS Processor blueprint has been cleaned up as the S3 PR includes the S3Event specifics, allowing this to be simplified.
2. Added a generic SNS to SQS blueprint.

**Comment by normj** [comment]
Merge to dev branch
